package.json

```diff
@@ -38,7 +38,9 @@
     "shelljs": "0.5.3",
     "underscore.string": "3.2.2",
     "wordwrap": "1.0.0",
+    "yeoman-assert": "2.1.1",
```

Comment:
yup, sorry i merged it so soon, I will update that

On Tue, Jan 12, 2016 at 3:39 PM, Pascal Grimaud <notifications@github.com>
wrote:

> In package.json
> <https://github.com/jhipster/generator-jhipster/pull/2612#discussion_r49422985>
> :
>
> > @@ -38,7 +38,9 @@
> >      "shelljs": "0.5.3",
> >      "underscore.string": "3.2.2",
> >      "wordwrap": "1.0.0",
> > +    "yeoman-assert": "2.1.1",
>
> My bad! I think it should be in devDependencies, instead of dependencies!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/2612/files#r49422985>
> .
>



-- 
thank you,
Deepu.k.s